app/views/shared/_alerts.html.erb

```diff
@@ -4,6 +4,9 @@
       <div class="alert alert-<%= klass %>">
         <button type="button" class="close" data-dismiss="alert">&times;</button>
         <%= message %>
+        <% if display_access_request_link?(type) %>
+          <%= link_to_request_access %>
+        <% end %>
```

Comment:
It's all a matter of taste and preference I think @sbrnunes. The code is actually generated in a helper and I did this deliberately because I, on the other hand, don't like to see Ruby conditionals in templates :)

I've seen this approach in a bunch of templates in Samson so I decided to go for something familiar here as well.

Also, I think it's easier to follow what's going on when you have one dedicated flash type in this case, instead of using 2 separate ones. Did you have a specific solution in mind to keep this code generic @sbrnunes?